### PR TITLE
SIGINT-553: assessment types

### PR DESCRIPTION
**Pull request by kiransynopsys**

### CODE REVIEW

File: test/contract/blackduck.e2e.test.ts [hunk 1]
```diff
@@ -122,9 +108,9 @@ export function mockBlackduckParamsExcept(blackduckConstants: string[]) {
 }
 
 export function setAllMocks() {
+  let blackduck: string[] = []
   jest.spyOn(configVariables, 'getWorkSpaceDirectory').mockReturnValue(__dirname)
-  jest.spyOn(validator, 'validatePolarisInputs').mockReturnValueOnce(true)
-
+  jest.spyOn(validator, 'validateBlackDuckInputs').mockReturnValueOnce(blackduck)
```
Comment (sig-tithi) [inline test/contract/blackduck.e2e.test.ts:113]: any reason we are changing from validateprolaris to validateblackduck?

File: test/contract/coverity.e2e.test.ts [hunk 2]
```diff
@@ -139,9 +125,9 @@ export function mockCoverityParamsExcept(coverityConstants: string[]) {
 }
 
 export function setAllMocks() {
+  let coverity: string[] = []
   jest.spyOn(configVariables, 'getWorkSpaceDirectory').mockReturnValue(__dirname)
-  jest.spyOn(validator, 'validatePolarisInputs').mockReturnValueOnce(true)
-
+  jest.spyOn(validator, 'validateCoverityInputs').mockReturnValueOnce(coverity)
```
Comment (sig-tithi) [inline test/contract/coverity.e2e.test.ts:130]: any reason for changing polaris to coverity?

File: test/contract/blackduck.e2e.test.ts [hunk 3]
```diff
@@ -34,23 +34,9 @@ describe('Blackduck flow contract', () => {
...
-    } catch (err: any) {
-      expect(err.message).toContain('failed with exit code 2')
-      error(err)
-    }
-  })
-
   it('With missing mandatory fields blackduck.api.token', async () => {
     mockBridgeDownloadUrlAndSynopsysBridgePath()
-    mockBlackduckParamsExcept(['BLACKDUCK_INSTALL_DIRECTORY', 'BLACKDUCK_SCAN_FAILURE_SEVERITIES', 'BLACKDUCK_API_TOKEN'])
+    mockBlackduckParamsExcept(['BLACKDUCK_INSTALL_DIRECTORY', 'BLACKDUCK_SCAN_FAILURE_SEVERITIES', 'BLACKDCK_API_TOKEN'])
```
Comment (kishorikumar) [inline test/contract/blackduck.e2e.test.ts:39]: BLACKDCK_API_TOKEN should be BLACKDUCK_API_TOKEN? Look at the blackduck spelling.

**Review by kishorikumar — CHANGES_REQUESTED** [review]

**Review by kishorikumar — CHANGES_REQUESTED** [review]
Make sure, we have ran corresponding workflow file in gitlab mirrored repo and it is working.

File: test/contract/blackduck.e2e.test.ts (hunk 3 above)
Comment (sig-tithi) [reply test/contract/blackduck.e2e.test.ts:39]: good catch!

File: test/contract/coverity.e2e.test.ts (hunk 2 above)
Comment (kiransynopsys) [reply test/contract/coverity.e2e.test.ts:130]: setting respective mocks for the validate methods.

File: test/contract/blackduck.e2e.test.ts (hunk 3 above)
Comment (kiransynopsys) [reply test/contract/blackduck.e2e.test.ts]: renamed it.

File: test/contract/blackduck.e2e.test.ts (hunk 1 above)
Comment (kiransynopsys) [reply test/contract/blackduck.e2e.test.ts:113]: mocking respective validate methods.

**Review by sig-tithi — APPROVED** [review]

**Review by kishorikumar — APPROVED** [review]

**Pull request merged**